src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/SetAzureRmDataLakeAnalyticsDataSource.cs

```diff
@@ -24,83 +24,39 @@ namespace Microsoft.Azure.Commands.DataLakeAnalytics
     [Alias("Set-AdlAnalyticsDataSource")]
     public class SetAzureDataLakeAnalyticsDataSource : DataLakeAnalyticsCmdletBase
     {
-        internal const string DataLakeParameterSetName = "Set a Data Lake storage account";
-        internal const string BlobParameterSetName = "Set a Blob storage account";
-
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
-            ParameterSetName = DataLakeParameterSetName, HelpMessage = "Name of the account to add the data source to.")
-        ]
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true,
-            ParameterSetName = BlobParameterSetName, HelpMessage = "Name of the account to add the data source to.")]
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true, 
+            HelpMessage = "Name of the account to update the data source in.")]
         [ValidateNotNullOrEmpty]
         [Alias("AccountName")]
         public string Account { get; set; }
 
         [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = true,
-            ParameterSetName = DataLakeParameterSetName,
-            HelpMessage = "The name of the Data Lake Store to add to the account.")]
-        [ValidateNotNullOrEmpty]
-        public string DataLakeStore { get; set; }
-
-        [Parameter(ValueFromPipelineByPropertyName = true, Position = 1, Mandatory = true,
-            ParameterSetName = BlobParameterSetName, HelpMessage = "The name of the Blob to add to the account.")]
+            HelpMessage = "The name of the Blob to update in the account.")]
         [ValidateNotNullOrEmpty]
         [Alias("AzureBlob")]
         public string Blob { get; set; }
 
         [Parameter(ValueFromPipelineByPropertyName = true, Position = 2, Mandatory = true,
-            ParameterSetName = BlobParameterSetName,
```

Comment:
@begoldsm removing the parameter sets from this cmdlet causes breaking changes. For this parameter, if a user has a script that was using the parameter set `DataLakeParameterSet`, it will no longer work because `AccessKey` is now a required parameter in every call of this cmdlet